miden-crypto/src/dsa/rpo_falcon512/keys/secret_key.rs

```diff
@@ -116,15 +117,17 @@ impl SecretKey {
     /// Signs a message with this secret key.
     #[cfg(feature = "std")]
     pub fn sign(&self, message: crate::Word) -> Signature {
```

Comment:
This no longer needs to be gated by `#[cfg(feature = "std")]`, right? My understanding is that One of the main benefits of making the scheme deterministic is that we don't need to draw the randomness from the OS, and so should be able to sign messages in WASM etc. pretty easily. Or do we still use something from `std` here?